README.md

```diff
@@ -98,17 +98,17 @@ The node already provides basic configurations for Sunny Boy Tripower 8.0/10.0 a
 
 ## How to find `message` and `value` id
 
-This section shows you how to get the `message id` and `value ids` the node needs in the input payload to read values from the devices.
+This section shows you how to get the `message id` and `value ids` the node needs in the input payload to read values from a device.
 
-1. connect to the webinterface of you device, i.e. `http(s)://192.168.1.42/#/login`
-2. got to the overview section of your device, i.e. `http(s)://192.168.1.42#/spotvalues`
+1. connect to the webinterface of your device, e.g. `http(s)://192.168.1.42/#/login`
+2. got to the overview section of your device, e.g. `http(s)://192.168.1.42#/spotvalues`
```

Comment:
There's also a slash missing in the URL.

Reply:
Thx, fixed it.